From bd323d82b489daa86112cf388b5499fd673ed479 Mon Sep 17 00:00:00 2001
From: blaizard <blaise.lengrand@gmail.com>
Date: Sat, 8 Jan 2022 15:31:54 +0100
Subject: [PATCH] Fixing number of registers to 172

---
 target/xtensa/core-esp32.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/xtensa/core-esp32.c b/target/xtensa/core-esp32.c
index 20f5644cd6..4525544d6a 100644
--- a/target/xtensa/core-esp32.c
+++ b/target/xtensa/core-esp32.c
@@ -27,6 +27,7 @@ static const XtensaOpcodeTranslators* esp32_opcode_translators[] = {
 static XtensaConfig xtensa_core_esp32 __attribute__((unused)) = {
     .name = "esp32",
     .gdb_regmap = {
+        .num_regs = 152,
         .reg = {
 #include "core-esp32/gdb-config.inc.c"
         }
-- 
2.25.1
